Dockerfile

```diff
@@ -19,8 +19,7 @@ RUN chmod -R 755 /app
 
 USER $APPLICATION_USER
 
-COPY --from=build /appbuild/build/libs/KtorEasy*all.jar /app/KtorEasy.jar
+COPY --from=build /appbuild/build/libs/KtorEasy.jar /app/KtorEasy.jar
 COPY --from=build /appbuild/resources/ /app/resources/
 WORKDIR /app
-
-CMD ["sh", "-c", "java -server -XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap -XX:InitialRAMFraction=2 -XX:MinRAMFraction=2 -XX:MaxRAMFraction=2 -XX:+UseG1GC -XX:MaxGCPauseMillis=100 -XX:+UseStringDeduplication -jar KtorEasy.jar"]
\ No newline at end of file
+ENTRYPOINT ["java","-jar","/app/KtorEasy.jar"]
```

Comment:
Changed to Entrypoint according to Ktor docs:
https://ktor.io/docs/docker-compose.html#configure-ktor-plugin